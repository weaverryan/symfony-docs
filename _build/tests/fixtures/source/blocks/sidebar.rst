
.. sidebar:: The sidebar's title

    some text inside sidebar
