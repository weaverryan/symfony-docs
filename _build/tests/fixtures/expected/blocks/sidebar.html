<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-wrapper">
    <div class="sidebar"></div>
    <div class="admonition admonition-sidebar">
        <p class="sidebar-title">The sidebar's title</p>
        <p>some text inside sidebar</p>
    </div>
</div>
</body>
</html>
